pyinstaller/requirements.in

```diff
@@ -2,5 +2,5 @@ pyinstaller==5.2
 pefile==2022.5.30
 macholib
 pywin32-ctypes
-babel==2.12.1
-pytz==2022.1
+# babel==2.12.1 should already be installed
+# pytz==2022.7.1 should already be installed
```

Comment:
delete

Reply:
fixed in [b8a3a7c](https://github.com/cryptoadvance/specter-desktop/pull/2446/commits/b8a3a7ce4ef1e1dfedd24e856d765754f1d5843f)